src/amo/components/Addon/index.js

```diff
@@ -286,6 +287,14 @@ export class AddonBase extends React.Component {
       }
 
       const description = addon.description ? addon.description : addon.summary;
+      // For any theme type, we want to hide the summary text here since that is
+      // already displayed in the header.
+      showAbout = !(
```

Comment:
Could we explicit this `!`? Maybe we could write `(condition ...) === false`?

Reply:
ahh sure..okay!